fix: use correct schema dialect for oas3.0.x and oas2.x (#2564)
Redocly/redocly-cli: .changeset/wicked-towns-rest.md

---
"@redocly/openapi-core": patch
"@redocly/cli": patch
---

> Fixed example validation for OpenAPI 3.0.x and OpenAPI 2.x by selecting the correct JSON Schema dialect.

Reviewer: ```suggestion
Fixed false positive errors in example validation for OpenAPI 3.0.x and OpenAPI 2.x.
```
